src/posts.fr-CA.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: layouts/posts.njk
+title: Articles
+lang: fr-CA
+translationKey: posts
+eleventyNavigation:
+  parent: fr-CA
+  key: Articles
+  order: 3
+permalink: "/fr/articles/{% if pagination.pageNumber > 0 %}page/{{ pagination.pageNumber + 1 }}/{% endif %}"
+pagination:
+  data: collections.posts_fr-CA
+  size: 10
+  alias: posts
+---
```

Comment:
There seems to be a lot of duplication between src/posts.fr-CA.md and src/posts.en-CA.md. I'd assume additional languages would also be similar. It may make sense to template these files as well at some point.

Reply:
I can look into this. Because the URL structure differs between the default language (no language slug in front) and other languages, there's a bit of differentiation that's needed so I had thought it would make sense to not try to template these at the risk of making it confusing for a site builder to figure out where all the post archive pages are coming from. But I will see what I can do.

Reply:
Just to clarify, I didn't mean this needed to happen in this PR, but something to think about for the future. If you think it make sense to do now, go for it, but we can also punt it for another time.

Reply:
Probably makes sense to punt it— it would open up a whole can of worms…